From: Reinhard Tartler <siretart@tauware.de>
Date: Fri, 12 Jul 2024 08:36:13 -0400
Subject: relax test

Upstream test assert that the git commit id is backed into the output,
which is not the case for the Debian build.
---
 test/001-basic.bats | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/001-basic.bats b/test/001-basic.bats
index bc30a17..20e0a3b 100644
--- a/test/001-basic.bats
+++ b/test/001-basic.bats
@@ -12,7 +12,7 @@ load helpers
     run_netavark version
     json="$output"
     assert_json "$json" ".version" =~ "^1\.[0-9]+\.[0-9]+(-rc[0-9]|-dev)?" "correct version"
-    assert_json "$json" ".commit" =~ "[0-9a-f]{40}" "shows commit sha"
+#    assert_json "$json" ".commit" =~ "[0-9a-f]{40}" "shows commit sha"
     assert_json "$json" ".build_time" =~ "20.*" "show build date"
     assert_json "$json" ".target" =~ ".*" "contains target string"
 }
